### DISCUSSION TITLE
Customized indicator

### DISCUSSION THREAD

**Discussion by jeff1228** [kernc/backtesting.py#1029] [Q&A]
Hi group,

I am new to coding, I am trading SPX, and want to create an indicator using VIX and VIX3M, which means the trading signal is not deprived from SPX itself, it's from VIX index. I came up with the code below, but seems not woking, can someone please take a look and shed some lights here?

def indicator():
  
    vratio = ta.hma(dfVIX.Close/dfVIX3M.Close, 3)
    return 

class SignalStrategy(Strategy):

    def init(self):
        self.mysignal = self.I(indicator)

    def next(self):
        if not self.position and crossover(self.mysignal,1):
        
                self.buy()
        
        elif crossover(1,self.mysignal):
                self.position.close()

Don't have a clue what's wrong!

**Comment by peacerider** [comment]
Is "ta" pandas-ta?
`vratio = ta.hma(dfVIX.Close/dfVIX3M.Close, 3)`

**Reply by BradKML** [reply]
Would like a follow-up to this for TA-Lib and Pandas-TA
